BaseTools/Conf/build_rule.template

```diff
@@ -108,6 +109,13 @@
 ## Build Rule Version Number
 #  build_rule_version=0.1
 #
+#
+# 1.00 - Increase this version tag any time you want user to get warning about updating this file in the Conf dir.  By default it does not do update existing conf dirs.
+# 1.01 - Add MASM for Microsoft ARM assembly files. "MS-ARMx-Assembly-Code-File.COMMON.COMMON"

```

Comment:
Were all of these changes included in this commit? For example, it seems `MS-ARMx-Assembly-Code-File.COMMON.COMMONc` may have already been here? If that's the case and everything is accounted for, what you have is fine, I just want to make sure you double checked these version entries against the file at this point in time.

Reply:
All the changes listed already exist in the file.  The `MS-ARMx-Assembly_Code-File.COMMON.COMMON` and `DEPS_FLAG` support we're added earlier but it's being listed in the version notes now.